### ISSUE TITLE
Bug: sync.afraid.org send 404

### ISSUE THREAD

**Issue by mrt2nbl** [qdm12/ddns-updater#449]
**TLDR**: *sync.afraid.org send a 404 page*

1. Is this urgent: No
2. DNS provider(s) you use: freedns
3. Program version: Running version latest built on 2023-02-02T00:41:38Z (commit 228fc50)

4. What are you using to run the container: docker-compose
5. Extra information (optional)

Logs:

```log


========================================
========================================
============= ddns-updater =============
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2023-02-02T00:41:38Z (commit 228fc50)

🔧 Need help? https://github.com/qdm12/ddns-updater/discussions/new
🐛 Bug? https://github.com/qdm12/ddns-updater/issues/new
✨ New feature? https://github.com/qdm12/ddns-updater/issues/new
☕ Discussion? https://github.com/qdm12/ddns-updater/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-03-30T09:58:28+02:00 INFO reading JSON config from file /updater/data/config.json
2023-03-30T09:58:28+02:00 DEBUG config read: {
  "settings": [
    {
      "provider": "duckdns",
      "host": "host",
      "token": "TOKEN",
      "ip_version": "ipv4",
      "provider_ip": true
    },
    {
      "provider": "freedns",
      "domain": "domain",
      "host": "host",
      "token": "token",
      "ip_version": "ipv4"
    }
  ]
}
2023-03-30T09:58:28+02:00 INFO Found 2 settings to update records
2023-03-30T09:58:28+02:00 INFO Reading history from database: domain duckdns.org host HOST
2023-03-30T09:58:28+02:00 INFO Reading history from database: domain mooo.com host HOST
2023-03-30T09:58:28+02:00 DEBUG configured to fetch IP: v4 or v6: false, v4: true, v6: false
2023-03-30T09:58:28+02:00 INFO [healthcheck server] listening on 127.0.0.1:9999
2023-03-30T09:58:28+02:00 INFO [backup] disabled
2023-03-30T09:58:28+02:00 INFO [http server] listening on :8000
2023-03-30T09:58:29+02:00 DEBUG your public IP address are: v4 or v6: <nil>, v4: MYIP, v6: <nil>
2023-03-30T09:58:29+02:00 DEBUG IPv4 address of HOST.duckdns.org is MYIP and your IPv4 address is MYIP, skipping update
2023-03-30T09:58:29+02:00 INFO IPv4 address of HOST.mooo.com is <nil> and your IPv4 address is MYIP
2023-03-30T09:58:29+02:00 INFO Updating record [domain: mooo.com | host: HOST| provider: freedns | ip: ipv4] to use MYIP
2023-03-30T09:58:29+02:00 DEBUG GET https://sync.afraid.org/u/TOKEN/ | headers: User-Agent: DDNS-Updater quentin.mcgaw@gmail.com
2023-03-30T09:58:30+02:00 DEBUG 404 Not Found | headers: Vary: Accept-Encoding; Date: Thu, 30 Mar 2023 07:58:52 GMT; Content-Type: text/html; charset=UTF-8; Connection: keep-alive; Server: nginx | body: Error 404 : Page not found
2023-03-30T09:58:30+02:00 ERROR bad HTTP status: 404: Error 404 : Page not found 

Configuration file (**remove your credentials!**):

```json

{
  "settings": [
    {
      "provider": "duckdns",
      "host": "HOST",
      "token": "TOKEN",
      "ip_version": "ipv4",
      "provider_ip": true
    },
    {
      "provider": "freedns",
      "domain": "mooo.com",
      "host": "HOST",
      "token": "TOKEN",
      "ip_version": "ipv4"
    } 
  ]
} 

Host OS:
Debian 11 unraid

Hi, it seem afraid move the update url to :
https://freedns.afraid.org/dynamic/update.php?TOKEN

[edit] I think it might be wrong and didn't activate v2 from freedns.
seems OK now.
